test-suite/js/src/test/scala/org/scalajs/testsuite/javalib/util/ArraysTest.scala

```diff
@@ -25,8 +24,9 @@ object ArraysTest extends ArraysTest
  */
 trait ArraysTest extends JasmineTest {
```

Comment:
It is preliminary work on the porting the `Array` and typed `TypedArray`s. I left these one for the next PR.